.travis.yml

```diff
@@ -14,12 +14,17 @@ env:
     - PATH="${PYENV_ROOT}/shims:${PATH}"
 
 script:
```

Comment:
This is confusing since - fwict it is unused except as an holder for anchors. If I'm right in that characterization, consider `x-script:` or `shared-scripts:` or some other non-overloaded name.

Reply:
I'm not right in that characterization - several matrix entries do inherit these scripts :/

I am a fan of:
```yaml
      script:
        - *test_1_14
        - *test_1_15_and_unspecified
```

Consider doing more of those and being explicit with the scripts for every matrix entry. This would have all scripts definied in one place up here - in x-scripts say - and referenced below explicitly in each job. Maybe. I just find these complex yaml configs very confusing to read. It may just be the nature of the beast.

Reply:
Good idea. I refactored the whole thing to take a similar approach, which has the benefit of making it much more explicit where we skip Py37 and will make it easier to update these entries as we release more stable versions.

Thanks for the suggestion.